jenkins/custom-ci/generic/custom-ci.groovy

```diff
@@ -80,6 +80,12 @@ pipeline {
                 description: 'Build motr rpm using dtm mode.'
         )
 
+        choice(
+            name: 'ENABLE_ADDB_PLUGIN',
+                choices: ['no', 'yes'],
```

Comment:
Default ENABLE_ADDB_PLUGIN is `yes` at https://github.com/Abhirup-A-G/cortx-re/blob/CORTX-31656-Updated/jenkins/custom-ci/generic/cortx-rgw-integration.groovy#L17 
We should do the same here also.

Reply:
Default needs to be no as requested by user. I have updated the code accordingly.